src/maluuba/xtd/optional.hpp

```diff
@@ -3,7 +3,7 @@
 
 #ifndef MALUUBA_XTD_OPTIONAL_HPP
 #define MALUUBA_XTD_OPTIONAL_HPP
-#if __cplusplus >= 201703L
+#if __cplusplus >= 201703L && __has_include(<optional>)
```

Comment:
`__has_include()` isn't a standard feature, might be good to guard this with `#ifdef __has_include`

Reply:
Just kidding it is standard :O  https://en.cppreference.com/w/cpp/preprocessor/include